subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/results/ReportGenerator.java

```diff
@@ -30,7 +31,7 @@
     public static void main(String... args) throws Exception {
         Class<?> resultStoreClass = Class.forName(args[0]);
         File outputDirectory = new File(args[1]);
-        ResultsStore resultStore = (ResultsStore) resultStoreClass.newInstance();
+        ResultsStore resultStore = (ResultsStore) ClassUtils.newInstance(resultStoreClass);
```

Comment:
Let's use `getConstructor().newInstance()` here, since we have control over the results store class.

Reply:
Sure.